app/pom.xml

```diff
@@ -638,6 +638,7 @@
               <exclude>**/*.webapp</exclude>
               <exclude>**/browserconfig.xml</exclude>
               <exclude>connector/**/META-INF/syndesis/connector/verifier/*</exclude>
+              <exclude>connector/**/META-INF/syndesis/connector/meta/*</exclude>
```

Comment:
Bit confusing as you did add license headers to some of these files. We should also take a look at why this wasn't caught by the license check we do on CI. I think it might be because if the files there not having an extension.